bigbluebutton-html5/imports/api/cursor/server/publishers.js

```diff
@@ -2,14 +2,15 @@ import Cursor from '/imports/api/cursor';
 import { Meteor } from 'meteor/meteor';
 import { check } from 'meteor/check';
 import Logger from '/imports/startup/server/logger';
-import { isAllowedTo } from '/imports/startup/server/userPermissions';
 
-Meteor.publish('cursor', (credentials) => {
-  // TODO: Some publishers have ACL and others dont
-  // if (!isAllowedTo('@@@', credentials)) {
-  //   this.error(new Meteor.Error(402, "The user was not authorized to subscribe for 'cursor'"));
-  // }
+import mapToAcl from '/imports/startup/mapToAcl';
 
+Meteor.publish('cursor', function() {
+  cursor = cursor.bind(this);
```

Comment:
Shouldn't there be a `const` or a `let` in front of `cursor`? Same for all the other publishers